Doc/library/typing.rst

```diff
@@ -1389,11 +1389,19 @@ These are not used in annotations. They are building blocks for declaring types.
    ``Point2D.__optional_keys__``.
    To allow using this feature with older versions of Python that do not
    support :pep:`526`, ``TypedDict`` supports two additional equivalent
-   syntactic forms::
+   syntactic forms. Firstly, using a literal :class:`dict` as the
+   second argument::
```

Comment:
I'd suggest just writing "a ``TypedDict`` may be created using a functional form". This parallels https://docs.python.org/3.10/library/enum.html#functional-api

Reply:
Thanks for this!